test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -297,9 +299,11 @@ public void UpdatesServiceNameFromIConfiguration()
                     new ConfigurationBuilder().AddInMemoryCollection(configuration!).Build());
             });
 
-        var zipkinExporter = new ZipkinExporter(new ZipkinExporterOptions());
+#pragma warning disable CA2000 // Dispose objects before losing scope
+        var zipkinExporter = new ZipkinExporter(new());
 
         tracerProviderBuilder.AddProcessor(new BatchActivityExportProcessor(zipkinExporter));
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
same question as above, can we use `using` here?

```suggestion
        using var zipkinExporter = new ZipkinExporter(new());

        tracerProviderBuilder.AddProcessor(new BatchActivityExportProcessor(zipkinExporter));
```

Reply:
Same, will be disposed by line 308 `using var provider = tracerProviderBuilder.Build();`